get_nxpRdLib.sh

```diff
@@ -3,18 +3,6 @@
 blue='\033[0;34m'
 NC='\033[0m' # No Color
 
-prereq() {
-  PREREQS="build-essential cmake $1"
-  
-  if dpkg -s $PREREQS > /dev/null 2>&1; then
-    echo -e "Prerequisites already installed: $PREREQS"
-  else
-    echo -e "[${blue}Installing prerequisites${NC}]"
```

Comment:
I'd prefer to see a message printed if missing dependencies are identified during the install.  I think that's the best of both worlds.  Prior to doing dependency installation in this script, I received many questions that boiled down to missing dependencies, even though the dependencies were clearly documented in the README.